core/pkg/resultshandling/printer/v2/prettyprinter/utils.go

```diff
@@ -27,6 +27,7 @@ var (
 	clusterScanRunText   = fmt.Sprintf("Run a cluster scan: %s", getCallToActionString("'$ kubescape scan'"))
 	installHelmText      = fmt.Sprintf("Install Kubescape in your cluster for continuous monitoring: %s", linkToHelm)
 	CICDSetupText        = fmt.Sprintf("Add Kubescape to your CI/CD pipeline: %s", linkToCICDSetup)
+	imagePatchText		 = fmt.Sprintf("Try patching your images with: %s", getCallToActionString("'$ kubescape patch'"))
```

Comment:
please note that this text is being printed even when running the `patch` command itself

Reply:
yes I noticed that, sorry about that, will fix this as well.